cluster/utils.sh

```diff
@@ -103,6 +103,7 @@ function create_cbt_table() {
     echo "Could not create table ${CBT_TABLE} on instance ${CBT_INSTANCE} in project ${PROJECT}"
     return 1
   fi
+  # TODO(sethtroisi): Figure out stuff here.
```

Comment:
Waiting for #566 then I'll add to the new file.

Reply:
Per our conversation earlier in the week, I believe that we want two tables with the very same schema (where both would have the new `e_` keyspace), but which are then combined by the trainer.

For the game-playing nodes, then, nothing needs to change in this file yet, because each node would be given a different setting for `CBT_TABLE` depending on its role, correct?

But since the trainer needs to identify both tables, maybe we will want some environment variable renaming.

Reply:
The trainer actually doesn't need to see the evaluation games at all -- these are games played with randomness off (no softpick, no dirichlet noise), which function as the way to evaluate which is a better model.  The resulting games are not actually used by the trainer; they're used by the overall pipeline code to control promotion of models, determine pairings, etc.